github-dark.css

```diff
@@ -415,7 +415,7 @@
   .inline-form .btn-plain, .pr-toolbar, .diffbar, .shade-gradient,
   #categories .list, .gist-banner, .select-menu-item-parent, .main-content,
   .showcase-page-header, .showcase-page-pattern:after, .description-inline,
-  .result-group a, .border-wrap, .select-menu-item, a.feed-icon {
+  .result-group a, .border-wrap, .select-menu-item, a.feed-icon, session-current {
```

Comment:
its easier if you click https://github.com/settings/security, since it is under security it contains sensitive info, mmmm 
session-device session-current should be it, I think my fix was only in a dream.

**this doesnt work.** Ill revert.


 screenshot

![capture](https://cloud.githubusercontent.com/assets/3521959/17114018/07ad1f1a-52a5-11e6-912d-f1f95ca5118e.PNG)

see the background shouldnt be grey like that should be dark or transparent?